src/components/Personalization/createComponent.js

```diff
@@ -47,7 +47,6 @@ export default ({
         const scopes = getDecisionScopes(renderDecisions, decisionScopes);
 
         if (!hasScopes(scopes)) {
-          showContainers();
```

Comment:
Help me understand this: If a customer only executes one `sendEvent` command and that command does not have scopes and `renderDecisions` is set to `false`, then the containers won't be shown until the timeout is hit? Is that what we want? Do we have tests that cover this?

Reply:
We need the prehiding snippet to be added only if bundle is loaded async, they care about flickering, and they have `sendEvent` with `renderDecisions: true`.
If they don't have a `sendEvent` command with `renderDecisions: true` on a page, they don't need this prehiding snippet.
If they load the bundle sync they also don't need the prehiding snippet, because all the prehiding/unhiding manipulations are done by us and they don't have to worry about it.

Reply:
Got it. Thanks.

Reply:
Can we make this clearer in [the documentation](https://docs.adobe.com/content/help/en/experience-platform/edge/solutions/target/flicker-management.html#managing-flicker-when-sdk-is-loaded-asynchronously)? Currently, it states:

> When the SDK is loaded asynchronously, it is required to use the prehiding snippet.

Reply:
yes I will, I have a special ticket for that.